tests/bootstrap.php

```diff
@@ -0,0 +1,29 @@
+<?php
+$cdash_root = dirname(dirname(__FILE__));
+chdir($cdash_root);
+
+function cdash_autoload($className)
+{
+    global $cdash_root;
+    $msg_dir =  "{$cdash_root}/include";
```

Comment:
Consider calling this `$include_dir` instead of `$msg_dir`

Reply:
Most def, changed.